RemotePlayWhatever/CMakeLists.txt

```diff
@@ -84,15 +86,17 @@ if(${SYSTEM_OS} STREQUAL "Linux")
     set_target_properties(RemotePlayWhatever PROPERTIES OUTPUT_NAME "remoteplaywhatever")
     install(TARGETS RemotePlayWhatever DESTINATION "bin")
 
-    file(READ "/etc/issue" ISSUE)
-    string(REGEX MATCH "Debian|Ubuntu" DISTRO ${ISSUE})
-    if(DISTRO MATCHES "Ubuntu" OR DISTRO MATCHES "Debian")
-        set(CPACK_GENERATOR "DEB")
-        set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Lets you force steam remote play together for any game")
-        set(CPACK_PACKAGE_DESCRIPTION "Tiny app that lets you force steam remote play together for any game")
-        set(CPACK_DEBIAN_PACKAGE_MAINTAINER "m4dEngi")
-        set(CPACK_DEBIAN_PACKAGE_DEPENDS "libwxbase3.0-0v5 (>= 3.0.4+dfsg), libwxgtk3.0-gtk3-0v5 (>= 3.0.4+dfsg)")
-        include(CPack)
+    if(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Linux")
+        file(READ "/etc/issue" ISSUE)
```

Comment:
Just adding the conditional statement here since `/etc/issue` doesn't exist on macOS, causing an error.